### ISSUE TITLE
Unable to post trades with Hitbtc

### ISSUE THREAD

**Issue by gpearson100** [DeviaVir/zenbot#952]
### System information
- **Have I written custom code (as opposed to using zenbot vanilla)**: **No**
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:  **Ubuntu 17.10**

- **Zenbot version**: **4.05**
- **NodeJS version**:  **v8.9.3**
- **Python version**:  **Python 2.7.14**
- **Exact command to reproduce**:
``./zenbot.sh trade --manual --period=1m hitbtc.ZEC-BTC --days=3 --buy_pct=100 --sell_pct=100 --debug
``
### Describe the problem
Unable to post trades with Hitbtc,  API access appears good as it gets correct account balance.  I also verified that my API key has post trade permission.
### Source code / logs
![image](https://user-images.githubusercontent.com/248626/34208052-7acb9d8c-e552-11e7-9392-1faaedb608eb.png)

**Comment by mtx250** [comment]
I believe remember that Hitbtc its in building, in the section of the code, the devolpeer write that this exchange haven't full funtionality.

https://github.com/carlos8f/zenbot/tree/master/extensions/exchanges/hitbtc

**Comment by DeviaVir** [comment]
This issue was moved to highvelocityspace/zenbot#26

**Comment by DeviaVir** [comment]
This issue was moved to highvelocityspace/zenbot#56

**Comment by station384** [comment]
I have run into this aswell.  I messed with the code, and got it to go further,  to the point there hitbtc complains about the amount ordered.  some of my solutions were to fix the products.json.   it doesn't include the the fields     min_size, max_size, min_total.  also not entirely sure what those values should match up to from the data hitbtc.

exchange.js also does not pass in the correct market to trade on.  confirmed this by hard coding a market.

I may keep hacking on this to get it to work,  but part of the problem  is information that hitbtc requires for the api is not available in the current config,    there would have to be mods the engine to accommodate some of the functions.   Not that big of a deal,  I just don't have much time to do it myself and post the changes.

In summary, it isn't finished, needs work.

**Comment by brucetus** [comment]
I've fixed placing buy/sell orders however now there are problems with the getOrder function

**Comment by station384** [comment]
I have reworked this exchange,  there is a pull #1346  that implements a working hitbtc.
